### DISCUSSION TITLE
Unable to Build Project in new Branch

### DISCUSSION THREAD

**Discussion by Anmol202005** [checkstyle/checkstyle#16047] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/104afd35-993e-494a-af33-41f785134121)

Not able to build project in a new branch but in the master branch the build is successful

**Comment by romani** [comment]
make sure there is no changes in master  branch to expect build to pass.

if no changes and still not passing please open html report of jacoco - https://github.com/checkstyle/checkstyle/wiki/How-to-run-certain-phases-and-validations#how-to-generate-ut-coverage-report and share it here, to spot what code lost coverage.

CI on master is green: https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=25007&view=logs&j=e663805c-ee13-5a24-f979-d725e0346c8e&t=40ff6e43-c5d7-5753-f947-23012624b0c6&l=1827

**Reply by Anmol202005** [reply, marked as answer]
Thanks Romani , the issue is resolved now.
